examples/event_handling/close_event.py

```diff
@@ -3,6 +3,7 @@
 Close Event
 ===========
 
+Show how to connect to close events
```

Comment:
add a period!

also, could you make it a bit more clear? E.g., "Connecting events that occur when the figure closes."